terraform/outputs.tf

```diff
@@ -29,7 +29,3 @@ output "container_instance_ecs_for_ec2_service_role_name" {
 output "ecs_service_role_name" {
   value = "${module.container_service_cluster.ecs_service_role_name}"
 }
-
-output "ecs_autoscale_role_arn" {
-  value = "${module.container_service_cluster.ecs_autoscale_role_arn}"
-}
```

Comment:
This was removed in [2.0.0](https://github.com/azavea/terraform-aws-ecs-cluster/blob/develop/CHANGELOG.md#200) of the AWS ECS Cluster module in favor of [using the default service-linked role](https://github.com/azavea/terraform-aws-ecs-cluster/pull/33).